CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## v1.2.1
+
+ENHANCEMENTS:
+
+- `resource/auth0_role`: Update role permission and permissions documentation ([#963](https://github.com/auth0/terraform-provider-auth0/pull/963))
+- `data-source/auth0_organization`: Refactor organization-related pagination method from `'Offset pagination'` to `'Checkpoint pagination'` ([#965](https://github.com/auth0/terraform-provider-auth0/pull/965))
```

Comment:
This wasn't done just for the data source, but for other resources as well related to organizations.